lib/actions/usbsdfuTargetActions.js

```diff
@@ -189,6 +189,16 @@ export function openDevice(selectedDevice) {
         .then(device => dispatch(loadDeviceInfo(device)));
 }
 
+// Reset device to Application mode
+export function resetDevice() {
+    return (dispatch, getState) => {
```

Comment:
This could be a promise based dispatch, since `.abort()` will be a promise (might even throw something).

Reply:
Sure.